include/ipc/ipc_service.h

```diff
@@ -203,6 +239,169 @@ int ipc_service_register_endpoint(const struct device *instance,
  */
 int ipc_service_send(struct ipc_ept *ept, const void *data, size_t len);
 
+/** @brief Get the TX buffer size
```

Comment:
Description of this function is ambiguous. How about something like
"Get the maximal size of a buffer which can be obtained now by ipc_service_get_tx_buffer()"